executor/ddl_test.go

```diff
@@ -211,6 +214,24 @@ func (s *testSuite) TestCreateDropTable(c *C) {
 	c.Assert(err, NotNil)
 }
 
+func (s *testSuite) TestCreateDropView(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	tk.MustExec("create or replace view drop_test as select 1,2")
+	_, err := tk.Exec("drop view if exists drop_test")
```

Comment:
Can I move 221 lines and 222 lines to 220 lines?

Reply:
Then following test will success, no error returns.
```go
	_, err = tk.Exec("drop view drop_test")
	c.Assert(err.Error(), Equals, "[schema:1051]Unknown table 'test.drop_test'")
```